modules/sdp/LocalSdpMunger.js

```diff
@@ -309,17 +309,53 @@ export default class LocalSdpMunger {
 
         if (audioMLine) {
             this._transformMediaIdentifiers(audioMLine);
+            this._injectSourceNames(audioMLine);
```

Comment:
Don't we want to do this only for Unified plan implementation of the client ?

Reply:
No. The plan-B client even if won't support sending more than one source at the same time, should still use the new signaling format.

Reply:
In that case, can we put this behind a flag so that we can have it disabled it on our prod environments. Also will older version of Jicofo throw a parsing error if it is talking to a new client ?

Reply:
I can put it behind the flag.

No, Jicofo will not throw an error because it's just one extra piece of information that it's going to include if available or ignore otherwise.